uniffi/src/cli.rs

```diff
@@ -79,6 +78,112 @@ enum Commands {
     },
 }
 
+// Type-bounds on trait implementations makes selecting between languages a bit tedious.
```

Comment:
I didn't get this docstring at first, I think it refers to the `match languages` statement which does seem tedious.  How about moving it to the match to make it more obvious.